src/net_processing.h

```diff
@@ -100,55 +100,27 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
                 CScheduler& scheduler, ChainstateManager& chainman, CTxMemPool& pool,
                 bool ignore_incoming_txs);
 
-    /**
-     * Overridden from CValidationInterface.
-     */
+    /** Overridden from CValidationInterface. */
     void BlockConnected(const std::shared_ptr<const CBlock>& pblock, const CBlockIndex* pindexConnected) override;
     void BlockDisconnected(const std::shared_ptr<const CBlock> &block, const CBlockIndex* pindex) override;
-    /**
-     * Overridden from CValidationInterface.
-     */
     void UpdatedBlockTip(const CBlockIndex *pindexNew, const CBlockIndex *pindexFork, bool fInitialDownload) override;
-    /**
-     * Overridden from CValidationInterface.
-     */
     void BlockChecked(const CBlock& block, const BlockValidationState& state) override;
-    /**
-     * Overridden from CValidationInterface.
-     */
     void NewPoWValidBlock(const CBlockIndex *pindex, const std::shared_ptr<const CBlock>& pblock) override;
 
-    /** Initialize a peer by adding it to mapNodeState and pushing a message requesting its version */
+    /** Implement NetEventsInterface */
     void InitializeNode(CNode* pnode) override;
-    /** Handle removal of a peer by updating various state and removing it from mapNodeState */
     void FinalizeNode(const CNode& node, bool& fUpdateConnectionTime) override;
-    /**
-    * Process protocol messages received from a given node
```

Comment:
Are you sure you want to lose all this documentation?

Reply:
Added a commit to move them into `NetEventsInterface`; that makes more of a conflict with #20864 so probably a good idea to get that reviewed and merged first.